src/scheduler/resv_info.c

```diff
@@ -915,6 +916,10 @@ check_new_reservations(status *policy, int pbs_sd, resource_resv **resvs, server
 	if (resvs == NULL)
 		return 0;
 
+	err = new_schd_error();
+	if (err == NULL)
+		return -1;
+
```

Comment:
Now that you allocate this and pass it to the dup functions, you need to free it someplace